pre-commit.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# No tests to update gh-pages
+BRANCH=$(git symbolic-ref HEAD 2>/dev/null)
+[ $BRANCH = "refs/heads/gh-pages" ] && exit 0
+
+git stash save -k -q
+make
```

Comment:
If xml2rfc had a `--validate-only` option, that would be ideal.  Let kramdown generate the XML, make sure it's valid with xml2rfc, and stop there with no files output.  Failing that, I'd guess that text-only is probably a little cheaper than HTML as well, but I don't know by how much.  (Tonight, I'm getting timeouts on all attempts to refresh references, so that dominates everything.)